.gitignore

```diff
@@ -1,12 +1,19 @@
 ### Unwanted Terraform files when using the examples ###
-
 examples/*/.terraform
 examples/*/terraform.tfstate
 examples/*/terraform.tfstate.*
 examples/*/.terraform.tfstate.lock.info
 examples/*/*-config
 examples/*/.terraform.lock.hcl
-
+examples/*/secrets.yml
+examples/*/terraform.tfstate*
+examples/*/.terraform.lock.hcl
+examples/*/kubeconfig.yml
+examples/*/issue.txt
+examples/*/log.txt
+examples/*/*.png
+examples/*/*.html
+examples/*/issuers.yml
```

Comment:
I think only you have this files in your folders. Should we add it to the `.gitignore` and include these exceptions for everybody?